dist/tools/riotboot_gen_hdr/Makefile

```diff
@@ -0,0 +1,39 @@
+RIOTBASE       := ../../..
```

Comment:
Please remove all the instances of `:=`

Reply:
Does it really hurts?

Reply:
Yes, because a few months from now, and nobody will know if the immediate assignments were necessary or not and if they can be safely removed.

And also (and this has nothing to do with the bootloader) it perpetuates the myth that `:=` is a sort of magical spell that one puts when things don't work.